src/main/java/seedu/address/logic/commands/ChangeGroupCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GROUP_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.group.Description;
+import seedu.address.model.group.Group;
+import seedu.address.model.group.GroupName;
+import seedu.address.model.student.Name;
+import seedu.address.model.student.Student;
+
+public class ChangeGroupCommand extends Command {
+
+    public static final String COMMAND_WORD = "changegroup";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Changes the group which the student belongs to"
+            + "by the students name to the new group name given.\n"
+            + "Parameters: " + PREFIX_NAME + "NAME " + PREFIX_GROUP_NAME + "GROUP NAME "
+            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "Brian Cheong "
+            + PREFIX_GROUP_NAME + "CS1231S";
+
+
+    public static final String MESSAGE_CHANGE_GROUP_SUCCESS = "Changed Student: %1$s to Group: %2$s";
+    public static final String MESSAGE_CHANGE_TO_SAME_GROUP = "The student is already in this group.";
+    public static final String MESSAGE_STUDENT_NONEXISTENT =
+            "The student does not exist. Please choose a student that exists.";
+    public static final String MESSAGE_GROUP_NONEXISTENT =
+            "The group indicated does not exist. Please create it first.";
+
+    private static final String PLACEHOLDER = "placeholder";
+    // TODO: may need to change this once change to groupname for student constructor
+    // Placeholder group for construction of student to find student.
+    private static final Group PLACEHOLDER_GROUP = new Group(new GroupName(PLACEHOLDER), new Description(PLACEHOLDER));
```

Comment:
It seems like `PLACEHOLDER` and `PLACEHOLDER_GROUP` are not used, we should remove it.